src/Utopia/Messaging/Adapters/Push/Pushy.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Utopia\Messaging\Adapters\Push;
+
+use Utopia\Messaging\Adapters\Push as PushAdapter;
+use Utopia\Messaging\Messages\Push;
+
+class Pushy extends PushAdapter
+{
+    /**
+     * @param  string  $secretKey The secret API  key that pushy provides.
+     */
+    public function __construct(
+        private string $secretKey,
+    ) {
+    }
+
+    /**
+     * Get adapter name.
+     *
+     * @return string
+     */
+    public function getName(): string
+    {
+        return 'Pushy';
+    }
+
+    /**
+     * Get max messages per request.
+     *
+     * @return int
+     */
+    public function getMaxMessagesPerRequest(): int
+    {
+        return 1000;
```

Comment:
Did you confirm from the documentation that this limit is 1000 for Pushy?

Reply:
@Diveshmahajan4 The limit here should be 100000